test.js

```diff
@@ -169,10 +169,23 @@ test('should deserialize array', t => {
 	deserializeNonError(t, [1]);
 });
 
+test('should deserialize empty object', t => {
+	deserializeNonError(t, {});
```

Comment:
`serializeError({})` was deserialized as `new Error('')`, now it throws a NonError